### ISSUE TITLE
known input formats aren't provided to the learner

### ISSUE THREAD

**Issue by kirsty-hames** [adaptlearning/adapt-contrib-textInput#162]
### Subject of the issue
Unless `_allowsAnyCase` or `_allowsPunctuation` are enabled, there's no mechanism to detect the input error.

For instance, if the correct answer is 'Kineo', answering 'kineo.' is marked incorrect when `_allowsAnyCase` or `_allowsPunctuation` are `false`, but correct when set to `true`.

Unless the content author notes that an answer is case sensitive or punctuation isn't allowed within the instruction text, a learner is unaware these are the reasons why they've answered incorrectly. 

Ref: [WCAG 3.3.3 Error Suggestion Level AA](https://www.w3.org/WAI/WCAG22/quickref/?versions=2.0#qr-minimize-error-suggestions)

### Expected behaviour
Upfront instruction should be included if answer is case sensitive or punctuation not allowed. Perhaps these could be default instructions provided by course globals?
